media/js/firefox/firstrun/firstrun.js

```diff
@@ -41,15 +41,22 @@
 
         skipbutton.onclick = onVerificationComplete;
 
-        Mozilla.Client.getFirefoxDetails(function(data) {
-            Mozilla.FxaIframe.init({
-                distribution: data.distribution,
-                gaEventName: 'firstrun-fxa',
-                onVerificationComplete: onVerificationComplete,
-                onLogin: onVerificationComplete,
-                onFormEnabled: disableSkipButton,
-                onNavigated:  hideOrShowSkipButton
-            });
+        Mozilla.UITour.getConfiguration('sync', function(config) {
```

Comment:
This throws an error `TypeError: Mozilla.UITour is undefined`. Please make sure to test your changes locally. Thanks!

Reply:
Thanks for review. I figure out that I need to run both `manage.py runserver` & `gulp` **at same time** to run bedrock properly.

Ref to `http://bedrock.readthedocs.io/en/latest/uitour.html#local-development`
While test please make sure to add default prefs in `browser/app/profile/firefox.js`
```
pref("browser.uitour.testingOrigins", "http://127.0.0.1:8000");
pref("browser.uitour.requireSecure", false);
```
and test the page on the custom firefox build.

Then open `http://127.0.0.1:8000/en-US/firefox/57.0/firstrun/` to see how it goes.
(without login sync -> normal signin page, with login to sync -> redirect to about:home directly after sunrise)

In local test I can't see the FxA iframe though, but it should be work.